src/Tasks/Copy.cs

```diff
@@ -215,7 +215,7 @@ private void LogDiagnostic(string message, params object[] messageArgs)
         {
             if (s_alwaysRetryCopy)
             {
-                Log.LogWarning(message, messageArgs);
+                Log.LogMessage(MessageImportance.Low, message, messageArgs);
```

Comment:
Pls adjust or remove the comment - as now it's out of sync with actual code.

Btw. can a `MessageImportance.High` be a better way to satisfy the need for visibility? Low importance message would not be sent to common loggers (Output windw, console) and hence it can confuse users that were used to see it there in case of problems.

Reply:
I am questioning the need for visibility. I don't think it has a value for people. They just need this file to be copied to destination.

Reply:
I do not have opinion here :-)
Sounds like you thought a bit about that - so I'm fine with your conclusion.
Let's remove the comment then.

Reply:
Low seems a reasonable default here - it will appear in binlogs and other locations, but not in a 'normal' `dotnet build` scenario.

Reply:
I am less sure about that. There are cases where copies need retries to work around infrastructure flakiness (copying to a network release share, for instance), but often retries are indicative of a bug in your build (overbuilding or racing to a file write). "Warning" seems like a pretty reasonable place to me (though I can empathize with wanting the warning to be suppressible).

Reply:
Now I have conflicting opinions from two most respected authorities in msbuild. I am fine with either, this is pros and cons:

a) low warning (as is ATM)
- pros: 
  - simple implementation
  - people would not see non translated slightly cryptic messages "Got {0} copying {1} to {2} and HR is {3}"
- cons: 
  -- might hide infrastructure flaky issues and/or bug in project or sln files causing these flakiness (mostly race conditions or redundant multi copies)

b) warnings (as was), but translated and suppressible. 
- pros: 
  - less probability of regression (it was this way for ages)
  - more visibly pointing on infrastructure flakiness or project files bugs
- cons: 
  - if we have some system problem which is causing those copy issues, people will have it more difficult to workaround it (MSBUILDALWAYSRETRY + nowarn:MSBXXXX)
  - when MSBUILDALWAYSRETRY and /WarnAsError infrastructure flakiness cascade into build flakiness

@baronfel  @rainersigwald please advise...

Reply:
I'm happy to keep warnings _as long as they get a code_ - we basically should never make a warning-level message without a suppressible code.

Reply:
Reworked into warnings. Ready for review